pybind_interface/pybind_main.cpp

```diff
@@ -54,6 +55,14 @@ Circuit<Cirq::GateCirq<float>> getCircuit(const py::dict &options) {
   }
 }
 
+NoisyCircuit<Cirq::GateCirq<float>> getNoisyCircuit(const py::dict &options) {
+  try {
+    return options["c\0"].cast<NoisyCircuit<Cirq::GateCirq<float>>>();
```

Comment:
Is the compiler smart enough to turn this into a move ? I would assume yes, but am not 100% sure.

Reply:
Not sure. This is a copy of the `getCircuit` method, though, so it should follow our pre-existing behavior.

Reply:
Is there a change requested here? Otherwise I'll mark this resolved.

Reply:
I think no change is requested here. It might be worth addressing this issue in a later PR.